prometheus_client/multiprocess.py

```diff
@@ -156,7 +157,6 @@ def mark_process_dead(pid, path=None):
     """Do bookkeeping for when one process dies in a multi-process setup."""
     if path is None:
         path = os.environ.get('PROMETHEUS_MULTIPROC_DIR', os.environ.get('prometheus_multiproc_dir'))
-    for f in glob.glob(os.path.join(path, f'gauge_livesum_{pid}.db')):
-        os.remove(f)
-    for f in glob.glob(os.path.join(path, f'gauge_liveall_{pid}.db')):
-        os.remove(f)
+    for mode in {m for m in Gauge._MULTIPROC_MODES if 'live' in m}:
```

Comment:
if we agree on the previous one then probably would want to make the same change here: `m.startswith('live')`